From 0155a533b1712a7f26e4dd9221dfd12874b2befa Mon Sep 17 00:00:00 2001
From: Johannes Schrimpf <johannes.schrimpf@blueye.no>
Date: Sun, 10 Mar 2019 01:39:37 +0100
Subject: [PATCH] remove deprecated and unused boost signals

---
 moveit_ros/planning/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/moveit_ros/planning/CMakeLists.txt b/moveit_ros/planning/CMakeLists.txt
index d66a6b0a8..05cca8f79 100644
--- a/moveit_ros/planning/CMakeLists.txt
+++ b/moveit_ros/planning/CMakeLists.txt
@@ -7,7 +7,7 @@ if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost REQUIRED system filesystem date_time program_options signals thread chrono)
+find_package(Boost REQUIRED system filesystem date_time program_options thread chrono)
 find_package(catkin REQUIRED COMPONENTS
   moveit_core
   moveit_ros_perception
-- 
2.19.1

